Blazor/Utils/HandleLanguageLink.razor

```diff
@@ -0,0 +1,15 @@
+﻿@inject IJSRuntime JS
+@inject NavigationManager Navigation
+
+@code {
+    protected override void OnInitialized()
+    {
+        JS.InvokeVoidAsync("cdtsBlazor.SetCurrentPage", Navigation.ToBaseRelativePath(Navigation.Uri));
```

Comment:
Is the lack of `await` here (and in event handler) intentional?

Reply:
Fixed (not intentional, i missed this class)